src/main/java/com/github/lunatrius/schematica/client/renderer/chunk/overlay/RenderOverlay.java

```diff
@@ -146,34 +165,45 @@ public void rebuildChunk(final float x, final float y, final float z, final Chun
         compiledOverlay.setVisibility(visgraph.computeVisibility());
     }
 
-    private int getSides(final IBlockState blockState, final Block block, final World world, final BlockPos pos, int sides) {
-        if (block.shouldSideBeRendered(blockState, world, pos, EnumFacing.DOWN)) {
+    private int getSides(final BlockType[][][] types, final int x, final int y, final int z) {
+        // The padding cannot be rendered (it lacks neighbors)
+        assert x > 0 && x < 17;
```

Comment:
Are these not compiled into the following code, which is not really what we'd want?
```java
if (!(x > 0 && x < 17))
    throw new AssertionError();
```